src/main/webapp/js/finscholar/collegepageview/collegepageview.js

```diff
@@ -18,12 +18,41 @@ goog.module('finscholar.collegepageview');
 
 const {CollegeDataHandler} = goog.require('datahandlers.collegepage');
 const {SinglePageView} = goog.require('finscholar.singlepageview');
-const {collegepage} = goog.require('finscholar.collegepageview.templates');
+const googDom = goog.require('goog.dom');
+const {collegepage, scholarshiplist} = goog.require('finscholar.collegepageview.templates');
+
+const SCHOLARSHIP_DIV_NAME = 'scholarships';
 
 /** Class for the college page view. */
 class CollegePageView extends SinglePageView {
   constructor() {
     super(new CollegeDataHandler(), collegepage);
+
+    /** @private {?Element} The container for scholarship buttons. */
+    this.scholarshipContainer_ = null;
+  }
+
+  /** @override */
+  async renderView() {
+    try {
+      await super.renderView();
+      this.scholarshipContainer_ = googDom.getElement(SCHOLARSHIP_DIV_NAME);
+      await this.renderScholarships_();
```

Comment:
We don't need a separate call to add links to colleges in scholarship page, so I did not change structure for singlepageview